client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -26,7 +26,12 @@ import getProfile from '../../helpers/APICalls/getProfile';
 import { useSnackBar } from '../../context/useSnackbarContext';
 import { CurrentProfile } from '../../interface/AuthApiData'
 
-export default function EditProfile(): JSX.Element {
+
+interface Props {
+  newUser: boolean
+}
+
+export default function EditProfile({newUser}: Props): JSX.Element {
```

Comment:
Rather than setting the new user as a boolean in the history object and passing it down as props, maybe we can access the user's profile through the context (auth context) ?